VMBackup/HandlerManifest.json

```diff
@@ -1,15 +1,14 @@
-[
-    {
-        "handlerManifest": {
-            "disableCommand": "main/handle.py -disable", 
-            "enableCommand": "main/handle.py -enable", 
-            "installCommand": "main/handle.py -install", 
-            "rebootAfterInstall": false, 
-            "reportHeartbeat": false, 
-            "uninstallCommand": "main/handle.py -uninstall", 
-            "updateCommand": "main/handle.py -update"
-        }, 
-        "name": "VMBackupForLinuxExtension", 
-        "version": "0.1.0.91"
-    }
-]
\ No newline at end of file
+[{
+    "namespace": "Microsoft.OSTCExtensions", 
+    "name": "VMBackupForLinuxExtension", 
+    "version": "0.1.0.993",
+    "handlerManifest": {
+        "disableCommand": "main/handle.py -disable", 
+        "enableCommand": "main/handle.py -enable", 
+        "installCommand": "main/handle.py -install", 
+        "rebootAfterInstall": false, 
+        "reportHeartbeat": false, 
```

Comment:
I alphabetized the the invocations of the script, followed by the boolean fields, followed by the enum fields (if any).